syntax = "proto3";
package mars.stn;


option java_package = "win.liyufan.im.proto";
option java_outer_classname = "ModifyGroupInfoRequestOuterClass";

//option objc_class_prefix = "LYF";

import "group.proto";
import "message_content.proto";

message ModifyGroupInfoRequest {
  GroupInfo group_info = 1;
  MessageContent notify_content = 2;
}
